src/applications/mhv-landing-page/sass/mhv-landing-page.scss

```diff
@@ -109,6 +109,11 @@ $anchor-font-size: 1.25rem; // == 20px
   visibility: hidden;
 }
 
+.custom-ordered-list {
+  margin-left: 0px;
+  padding-left: 24px;
```

Comment:
I'm not sure we need to customize the padding of a list. I'd push back on that with UX (or confirm that this was intended)

If we must do this, I'd suggest following the BEM style naming, so something like `mhv-c-custom-list`.

I'd also want to see this in `em` values since we'd want this to scale when folks zoom